src/Microsoft.Health.Fhir.Core/Configs/ExportJobConfiguration.cs

```diff
@@ -15,6 +15,23 @@ public class ExportJobConfiguration
         /// </summary>
         public bool Enabled { get; set; }
 
+        /// <summary>
```

Comment:
This will be a breaking change so we need to be explicit about how we want to handle it.

If this code is deployed and there is an existing export job record, the new code will fail, which is fine if that's what we decide to do. Just want to make sure we think about breaking change scenarios when changing existing code.